packages/kit/src/views/Earn/pages/EarnProtocolDetails/index.tsx

```
@@ -0,0 +1,727 @@
+        screen: EModalStakingRoutes.ManagePosition,
+        params: {
+          networkId,
+          symbol,
+          provider,
+          vault,
+          tab,
+          tokenImageUri: tokenInfo?.token?.logoURI,
+        },
+      });
+    },
+    [appNavigation, networkId, symbol, provider, vault, tokenInfo?.token?.logoURI],
```

ESLint

[error] Replace `appNavigation,·networkId,·symbol,·provider,·vault,·tokenInfo?.token?.logoURI` with `⏎······appNavigation,⏎······networkId,⏎······symbol,⏎······provider,⏎······vault,⏎······tokenInfo?.token?.logoURI,⏎····`

(prettier/prettier)